tests/sentry/api/endpoints/test_project_group_index.py

```diff
@@ -1455,3 +1446,45 @@ def test_delete_by_id(self):
 
         assert Group.objects.filter(id=group4.id).exists()
         assert GroupHash.objects.filter(group_id=group4.id).exists()
+
+    def test_bulk_delete(self):
+        groups = []
+        for i in range(10, 41):
+            groups.append(
+                self.create_group(
+                    project=self.project,
+                    checksum=six.binary_type(i) * 16,
+                    status=GroupStatus.RESOLVED))
+
+        for group in groups:
+            GroupHash.objects.create(
+                project=group.project,
+                hash=uuid4().hex,
+                group=group,
+            )
+
+        self.login_as(user=self.user)
+
+        # if query is '' it defaults to is:unresolved
+        url = self.path + '?query='
+        response = self.client.delete(url, format='json')
+
+        assert response.status_code == 204
+
+        for group in groups:
+            assert Group.objects.get(id=group.id).status == GroupStatus.PENDING_DELETION
+            assert not GroupHash.objects.filter(group_id=group.id).exists()
+
+        Group.objects.filter(
+            id__in=[
+                group.id for group in groups]).update(
+            status=GroupStatus.UNRESOLVED)
```

Comment:
@ceorourke why do you update the rows here? I can't figure it out (I'm currently looking at this code base to support deleting issue platform issues)

Reply:
I truly have no recollection of this 6 year old PR, but looking at this comment in another test in this file: https://github.com/getsentry/sentry/blob/master/tests/snuba/api/endpoints/test_project_group_index.py#L1519-L1519 I'm guessing that's how it used to work

Reply:
thanks @ceorourke